state/runtime/evm/state.go

```diff
@@ -44,10 +44,10 @@ var (
 	errInvalidJump           = errors.New("invalid jump destination")
 	errOpCodeNotFound        = errors.New("opcode not found")
 	errReturnDataOutOfBounds = errors.New("return data out of bounds")
+	errInvalidGasValue       = errors.New("invalid gas value")
```

Comment:
In that case, `invalid gas value` is misleading, because it is not the `gas` parameter that is in question but the `balance`. That's why I was suggesting change the error message to `invalid balance value`.